lib/duffel_api/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DuffelAPI
-  VERSION = "0.4.1"
+  VERSION = "0.6.0"
```

Comment:
see comment from my other PR: we want to tag and release new versions after merging stuff into `main`